camunda-formio-tasklist-vue/src/services/FormatTime.ts

```diff
@@ -6,15 +6,13 @@ export const getISODateTime = (date?: Date | string) => {
   }
 };
 
-export const getFormattedDateAndTime = (date: Date) => {
-  return new Date(date).toLocaleDateString("en-us", {
-    weekday: "long",
-    year: "numeric",
-    month: "long",
-    day: "numeric",
+export const getFormattedDateAndTime = (date: Date)=>{
+  return new Date(date).toLocaleDateString('en-us',  { 
+    weekday: 'long', year: 'numeric', month: 'long', day: 'numeric',hour: '2-digit', minute: '2-digit', hour12: true
```

Comment:
split it into separate lines for readability

Reply:
done

Reply:
this issue fixed

Reply:
Ok